src/block/carousel/frontend-carousel.js

```diff
@@ -588,6 +590,37 @@ class _StackableCarousel {
 
 		this.hasTouched = false
 	}
+
+	// Pause autoplay while inline navigation is in progress
+	fixInlineScrollNavigation = () => {
+		let listenerTimeout = null
+		let fallbackTimeout = null
+
+		const handleEndScroll = () => {
+			clearTimeout( fallbackTimeout )
+			this.unpauseAutoplay()
+			document.removeEventListener( 'scroll', scrollListener )
+		}
+
+		// After 100ms of non-scrolling, then the navigation scroll is done, restart the autoplay
+		const scrollListener = throttle( () => {
+			clearTimeout( listenerTimeout )
+			listenerTimeout = setTimeout( handleEndScroll, 100 )
+		}, 80 )
+
+		// Listen to any scroll navigation clicks, including dynamically added
+		document.addEventListener( 'click', e => {
+			// Check if we're inside an anchor link
+			// eslint-disable-next-line @wordpress/no-global-event-listener
+			if ( e.target.closest( '[href^="#"]' ) ) {
+				this.pauseAutoplay()
+				document.addEventListener( 'scroll', scrollListener, { passive: true } )
+				// Fallback to autoplay is resumed even if the user manually override
+				// the inline navigation or by clicking the link again
+				fallbackTimeout = setTimeout( handleEndScroll, 1000 )
```

Comment:
I think you can also just do the same `listenerTimeout` here 
```suggestion
				clearTimeout( listenerTimeout )
				listenerTimeout = setTimeout( handleEndScroll, 1000 )
```